icgc-argo/argo-data-submission
# Argo data submission wf@1.2.0

Addressing:
https://github.com/icgc-argo/argo-data-submission/issues/96

To test:
- with skipped tests
```
cd argo-data-submission/argo-data-submission-wf/tests

nextflow run ../../main.nf \
-profile docker,rdpc_qa \
-params-file local-test-job-7-InputCramJson.json \
--api_token ${API_TOKEN} \
--skip_duplicate_check true \
--ref_genome_fa ../../test_data/tiny-grch38-chr11-530001-537000.fa.gz \
--score_cpus 1 --score_mem 2 \
--skipping_tests 'c681,c683'
```
- without
```
cd argo-data-submission/argo-data-submission-wf/tests

nextflow run ../../main.nf \
-profile docker,rdpc_qa \
-params-file local-test-job-7-InputCramJson.json \
--api_token ${API_TOKEN} \
--skip_duplicate_check true \
--ref_genome_fa ../../test_data/tiny-grch38-chr11-530001-537000.fa.gz \
--score_cpus 1 --score_mem 2
```

Running the follow on old version [1.1.0](https://github.com/icgc-argo/argo-data-submission/releases/tag/1.1.0) should result in the following
```
No such file: argo-data-submission-wf/tests/NO_FILE4
```

New version should complete without issue.
